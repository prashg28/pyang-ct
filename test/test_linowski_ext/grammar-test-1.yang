module grammar-test {

   namespace "urn:org:example:xml:ct:grammar-test";
   prefix "gr";

   import complex-types { prefix "ct"; }

   // ct nested into a module
   ct:complex-type ct1;

   ct:instance i1 {
   }

   container c1 {
       // ct nested into a container
       ct:complex-type ct2;
   }

   list l1 {
       // ct nested into a list
       ct:complex-type ct3;

       leaf leaf1 {
           type int8;
       }

       key leaf1;
   }

   rpc rpc1 {
       // ct nested into an rpc
       ct:complex-type ct4;
       input {
           // ct nested into an input
           ct:complex-type ct5;
           leaf leaf2 {
               type int8;
           }
       }

       output {
           // ct nested into an output
           ct:complex-type ct6;
           leaf leaf3 {
               type int8;
           }
       }
   }

   notification n1  {
       // ct nested into a notification
       ct:complex-type ct6;
   }
}
