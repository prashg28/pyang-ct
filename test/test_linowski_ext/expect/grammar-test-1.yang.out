grammar-test-1.yang:11: error: EXPECTED_KEYWORD_2
grammar-test-1.yang:16: warning: UNUSED_COMPLEX_TYPE
grammar-test-1.yang:21: warning: UNUSED_COMPLEX_TYPE
grammar-test-1.yang:32: warning: UNUSED_COMPLEX_TYPE
grammar-test-1.yang:35: warning: UNUSED_COMPLEX_TYPE
grammar-test-1.yang:43: warning: UNUSED_COMPLEX_TYPE
grammar-test-1.yang:52: warning: UNUSED_COMPLEX_TYPE
